android/src/com/frostwire/android/gui/adapters/menu/ResumeDownloadMenuAction.java

```diff
@@ -32,9 +42,10 @@
  * @author gubatron
  * @author aldenml
  */
-public final class ResumeDownloadMenuAction extends MenuAction {
+public final class ResumeDownloadMenuAction extends MenuAction implements AbstractDialog.OnDialogClickListener {
 
     private final BittorrentDownload download;
+    private static final String DLG_TURN_BITTORRENT_BACK_ON = "DLG_TURN_BITTORRENT_BACK_ON";
```

Comment:
let's make this a dummy `Object` and save memory.

Reply:
`private static final Object DLG_TURN_BITTORRENT_BACK_ON = new Object()`

Reply:
ok, since `onDialogClick(String tag)` does take a String, I suppose we can't do that.
However we can just put a single character on the string instead:
`private static final String DLG_TURN_BITTORRENT_BACK_ON = "0";`